packages/wdio-firefox-profile-service/src/launcher.ts

```diff
@@ -15,7 +17,8 @@ export default class FirefoxProfileLauncher {
         }
 
         if (this.options.profileDirectory) {
-            this.profile = await promisify(Profile.copy)(this.options.profileDirectory)
+            const profileCopy : (options: WebdriverIO.ServiceOption | string) => Promise<any> = promisify(Profile.copy)
```

Comment:
I don't think a `Promise<any>` is returned here, it should be a Firefox options profile.

```suggestion
            const profileCopy: (options: WebdriverIO.ServiceOption | string) => Promise<any> = promisify(Profile.copy)
```